src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Noticia+Text:ital,wght@0,700;1,400;1,700&family=Roboto:wght@500&display=swap");
```

Comment:
for better performance it's recommended to import styles in html using the link tag